src/App.js

```diff
@@ -1,12 +1,21 @@
-import React from 'react';
+import React, { Component } from "react";
+import DisplayAllArticles from "./Components/DisplayAllArticles";
+import {Container, Header,Divider} from 'semantic-ui-react';
 
-
-function App() {
-  return (
-    <div>
-     <h1>GNOSIS</h1>
-    </div>
-  );
+class App extends Component {
```

Comment:
Why did you refactor this to a class component?

Reply:
I had a discussion with Seth about using `function` components and `class` components, after he had talked with a coach, if I remember correctly.  From that, I gathered that we will just use `class` components for consistency and simplicity since that is the style we worked with before.  As far as I understand on the technical side, with what we are doing in this project, the effect of using either type is more or less the same so the change is mainly for consistency.  I'm fine with using `function` components this if there is a better technical reason.

Reply:
Functional components are the default. Use class components if you want to use state (you could however still use functional components with state using hooks)

Reply:
Fixed.